src/main/java/com/sparta/popupstore/domain/popupstore/controller/PopupStoreController.java

```diff
@@ -63,6 +69,30 @@ public ResponseEntity<PopupStoreFindOneResponseDto> getPopupStoreFindOne(@PathVa
         return ResponseEntity.ok(popupStoreService.getPopupStoreFindOne(popupStoreId));
     }
 
+    @Operation(summary = "팝업 스토어 단건 조회", description = "팝업스토어 단건조회(상세보기)")
+    @GetMapping("/popupstores/{popupStoreId}")
+    public ResponseEntity<PopupStoreFindOneResponseDto> getPopupStoreFindOne(
+            @PathVariable Long popupStoreId,
+            @AuthUser User user,
+            HttpServletRequest request,
+            HttpServletResponse response
+    ) {
+        String cookieName = "viewedPopup_" + popupStoreId;
+
+        Cookie cookie = WebUtils.getCookie(request, cookieName);
+        if (cookie == null) {
+            popupStoreService.viewPopupStore(popupStoreId);
+
+            ResponseCookie responseCookie = ResponseCookie.from(cookieName, "true")
+                    .path("/")
+                    .maxAge(60 * 60 * 24)
+                    .build();
+
+            response.addHeader(HttpHeaders.SET_COOKIE, responseCookie.toString());
```

Comment:
```suggestion
            ResponseCookie responseCookie = ResponseCookie.from(cookieName, "true")
                    .path("/")
                    .maxAge(VIEW_TOKEN_LIFETIME)
                    .build();

            response.addHeader(HttpHeaders.SET_COOKIE, responseCookie.toString());
```

상수 값은 따로 선언해두는 것이 좋을 것 같습니다.

Reply:
```suggestion
            webUtil(cookieName, response);
```
그리고 쿠키를 만드는 로직이니 controller가 아니라 webUtil에서 담당하는 것이 좋을 것 같고요.

Reply:
수정했습니다!